src/tracing/log_declares.h

```diff
@@ -27,6 +27,7 @@ struct MessageReceivedInfo;
 struct NodeLookupInfo;
 struct NodeDiscoveredInfo;
 struct NodeDiscoveryFailedInfo;
+class MetricEvent;
```

Comment:
why is this here and actual definition included in metric_event.h?

Reply:
Following pattern for "data logging" as in the struct's above